wordpress-mobile/WordPress-Android
# Issue/13326 threat details & history design review

Parent #13326

Threat Details & History Screens Design Review PR

Threat Details

Current Threat | Fixed Threat | Ignored Threat | Dark Mode
------------|-------------|-------------|------------
![light mode](https://user-images.githubusercontent.com/1405144/106723858-8ff9b400-662d-11eb-85c1-4ac532787d8b.png) |  ![fixed](https://user-images.githubusercontent.com/1405144/106724484-478ec600-662e-11eb-9c16-c370c4289a0c.png) | ![ignored](https://user-images.githubusercontent.com/1405144/106724541-570e0f00-662e-11eb-8aff-1a2cda947e98.png)|  ![dark_mode](https://user-images.githubusercontent.com/1405144/106723876-938d3b00-662d-11eb-800b-5c9736c6a8fe.png)


History

Loading | Fixed | Ignored | Error | Dark Mode
---------|------|---------|------|------------
![history_loading](https://user-images.githubusercontent.com/1405144/106738264-b6bfe680-663d-11eb-918b-8594f44eb358.png) | ![history_fixed](https://user-images.githubusercontent.com/1405144/106738289-bf182180-663d-11eb-947f-352710d1e03b.png) | ![history_ignored](https://user-images.githubusercontent.com/1405144/106738315-c7705c80-663d-11eb-95da-65c87c1919c3.png) | ![history_something_went_wrong](https://user-images.githubusercontent.com/1405144/106738423-e8d14880-663d-11eb-8d5f-1b1f6308525e.png) |![history_dark_mode](https://user-images.githubusercontent.com/1405144/106738351-d1925b00-663d-11eb-95a6-6be16ef10380.png)

**Testing**

Prerequisite:

- Make sure both Scan feature flag and MySiteImprovements flag are set to on in the App settings.
- You have access to WP.com account with a site having scan capability and multiple threats (e.g. http://do.wpmt.co/jp-complete).

PS -  Jetpack Threat Tester (Internal ref: pbuNQi-Kb) can be used to re-add threats once they're fixed. 

Merge Instructions:

1. Make sure PR #13949 is merged to develop
2. Remove "Not Ready for Merge" label
3. Merge this PR

**Notes**

- "DiffViewer" for a core file threat is not implemented.
- Tweaks for tablet - not included in this MVP release. 
- History screen does not include date headers.

PR submission checklist:
